test/TextToBinary.cpp

```diff
@@ -62,6 +62,27 @@ TEST(GetWord, Simple) {
   EXPECT_EQ("abc", AssemblyContext(AutoText("abc\n"), nullptr).getWord());
 }
 
+TEST(GetWord, MultipleWords) {
```

Comment:
This tests more than just multiple words -- it tests various forms of word break.  I'd split it into multiple tests with separate responsibilities.